app/views/provider/admin/dashboards/widgets/_service_top_traffic.html.slim

```diff
@@ -23,10 +23,11 @@ article.DashboardWidget id = widget.id
     - else
       p
         ' In order to show Top Applications you need to have at least one application sending traffic to the #{friendly_service_name(service)}.
-      p
-        ' Consider
-        => link_to 'making some test calls', edit_admin_service_integration_path(widget_service_id, :anchor => 'staging')
-        | from the Integration page to get a feel for what you'd see here.
+      - unless apiap? 
```

Comment:
so instead of hiding everything the link should be changed

Reply:
Pointing to `apiconfig/services/2/integration` ?

Reply:
`admin_service_integration_path(widget_service_id)`

Reply:
(we should also tell them to promote config to staging if it has not been done but lets leave that for a later iteration)